### ISSUE TITLE
Initial content in tab pane doesn't fade in 

### ISSUE THREAD

**Issue by tomkel** [twbs/bootstrap#17717]
[According to the docs](http://getbootstrap.com/javascript/#tabs-usage), one should add the class `.in` to a tab pane to fade in initial content. However, this doesn't happen.

v3: http://jsbin.com/yiyidawipo/1/edit?html,output
v4: http://jsbin.com/pejehozisa/1/edit?html,output

Instead, "Hello, world!" appears without any initial fade-in.

The `.in` definition appears in component-animations.less as 
```css
.fade {
  opacity: 0;
  .transition(opacity .15s linear);
  &.in {
    opacity: 1;
  }
}
```

This doesn't work for the initial content, as shown above. 
However the following example shows that changing the opacity via javascript does allow the initial content to fade in correctly.
http://jsfiddle.net/SO_AMK/a9dnW/

**Comment by twbs-lmvtfy** [comment]
Hi @tomkel!

You appear to have posted a live example (http://fiddle.jshell.net/SO_AMK/a9dnW/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 8, column 3: [W005](https://github.com/twbs/bootlint/wiki/W005): Found what might be an outdated version of jQuery; Bootstrap requires jQuery v1.9.1 or higher

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by tomkel** [comment]
I have achieved the desired functionality by designating no "active" tab in the HTML, then calling `tab('show')` after everything has loaded.

http://jsbin.com/ruvurodaci/1/edit?html,js,output

**Comment by cvrebert** [comment]
This is working as expected, the docs' phrasing just isn't the clearest. The majority of users don't want a fade effect in the scenario you describe.

**Comment by cvrebert** [comment]
Fixed by #17726 & #17727.
